MapboxCoreNavigation/RouteController.swift

```diff
@@ -188,7 +194,6 @@ open class RouteController: NSObject {
     /// updateRouteLeg is used to notify nav-native of the developer changing the active route-leg.
     private func updateRouteLeg(to value: Int) {
         let legIndex = UInt32(value)
-        navigator.changeRouteLeg(forRoute: 0, leg: legIndex)
```

Comment:
This redundant call to `MBNavigator.changeRouteLeg(forRoute:leg:)` was introduced in c5ee2acaacbb845f20294c2a50014c609938853a for #2138. It looks like an oversight to me too, rather than a workaround for something.